tests/vmi_lifecycle_test.go

```diff
@@ -2039,6 +2070,17 @@ func waitForVMIRebooted(vmi *v1.VirtualMachineInstance, login console.LoginToFun
 	}, 300)).To(Succeed(), "expected reboot record")
 }
 
+func waitForVMIReset(vmi *v1.VirtualMachineInstance) error {
+	By(fmt.Sprintf("Waiting for vmi %s reset", vmi.Name))
+	if vmi.Namespace == "" {
+		vmi.Namespace = testsuite.GetTestNamespace(vmi)
+	}
```

Comment:
This should not be necessary when using the VMI returned by `libvmops.RunVMIAndExpectLaunch`.

Reply:
I'd prefer to inline this helper into the test case.

Reply:
This is following the same precedence established by the function right above this one.